packages/ui/src/components/navigation/header-navigation/index.tsx

```diff
@@ -67,6 +68,29 @@ const messages = defineMessages({
 	},
 })
 
+const Z3USLogoLink = () => {
+	const navigate = useNavigate()
+
+	const handleZ3USLogoClick = () => {
+		const currentURL = window.location.href
+		const isZ3USDomain = currentURL.includes('z3us')
```

Comment:
window.location.hostname === 'z3us.com'
```suggestion
		const currentURL = window.location.hostname
		const isZ3USDomain = window.location.hostname === 'z3us.com'
```